src/components/presentational/SingleExternalAccount/SingleExternalAccount.css

```diff
@@ -17,17 +17,20 @@
     top: 0;
     cursor: pointer;
     padding: 16px;
-    color: #bbb;
-    width:auto;
+    color: var(--mdhui-text-color-4);
+    width: auto;
     font-size: .88em;
-    border:none;
-    background:none;
-    height:100%;
+    border: none;
+    background: none;
+    height: 100%;
 }
 
 .mdhui-single-external-account .external-account-header .external-account-provider-logo {
     height: 24px;
     display: block;
+    background: #FFF;
```

Comment:
Move to a var maybe?

Reply:
This is intentionally hardcoded since this is an image background